src/test/js/shared/accessibility/runner.js

```diff
@@ -0,0 +1,87 @@
+const HTMLCS = require('html_codesniffer');
+const fs = require('fs');
+const testConfig = require('../../config/test_data.js');
+
+const result = {
+  PASSED: 'passed',
+  FAILED: 'failed',
+};
+
+const resultObj = {
+  appName: 'FPL',
```

Comment:
Need to be changed to **idam-web-public**

Reply:
This has been updated now. Thank you!